### PR TITLE
Feature: load environment variables in poe command

### LINKED ISSUE #150: Load environment variables in poe command
## User stories
As a developer I want to be sure that the environment variables are loaded correctly when executing tests or interacting with the database. 
Therefore replace loading the environment environments with `dotenv` when running tests or using the script `db.py` with `envfile=` field in the poe command

## (Wiki) References
- [preferred way](https://github.com/BP2022-AP1/bp2022-ap1/wiki#environment-variables)

## Acceptance Criteria
- [x] remove `dotenv` call in `conftest.py`
- [x] remove `dotenv` call in `config.py`
- [x] remove dependency dotenv
- [x] add `envfile=` field to tests

### PR DESCRIPTION
**Pull request by antonykamp**
Fixes #150 

Replaces loading the environment variables with dotenv with loading it with poe. Now commands, like `poe db recreate`, get the environment variables too.

## PR checklist

- [x] Acceptance criteria fulfilled
- [x] Additional features are tested
- [x] Docs (code, wiki & diagrams) updated
- [x] Breaking changes anounced
- [x] Dev-branch has been merged into local branch to resolve conflicts
- [x] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

**Review by instant-sky — APPROVED** [review]
Cool, looks like some serious downsizing!

File: src/config.py [hunk 1]
```diff
@@ -1,4 +0,0 @@
-from dotenv import load_dotenv
-
-load_dotenv(".env.shared")
-load_dotenv(".env.secret")
```
Comment (SaturnHafen) [inline src/config.py:4]: Is there documentation on what env vars are available?

Comment (antonykamp) [reply src/config.py:4]: You can find them in https://github.com/BP2022-AP1/bp2022-ap1/blob/dev/.env.secret.sample 😇

Comment (SaturnHafen) [reply src/config.py:4]: Well then you need to change the file to say that it specifies where to add the env-vars. (They are not loaded from the file anymore, correct?)

Comment (antonykamp) [reply src/config.py:4]: They are still loaded from a file 😇 We're going to add a poe command to run the server. Then we'll specify `.env.shared` and `.env.secret` as env-files in the command 👍

Comment (SaturnHafen) [reply src/config.py:4]: Is there an issue for that?

Comment (antonykamp) [reply src/config.py:4]: see #99

**Review by SaturnHafen — APPROVED** [review]

**Comment by SaturnHafen** [comment]
Please tick the requirement-boxes in the linked issue :)

**Pull request merged**